Update in line 44 to correctly refer app type

ebbypeter opened the pull request:
Line 44 mentioned monolithic application instead of micro service application. Updating to avoid confusion when reading the doc.

azurecla commented:
Hi __@gronster__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

gronster commented:
Edit complete

ebbypeter committed:
Update in line 44 to correctly refer app type

Line 44 mentioned monolithic application instead of micro service application. Updating to avoid confusion when reading the doc.
